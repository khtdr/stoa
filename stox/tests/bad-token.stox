#!/usr/bin/env stox

âš¡// no such token

// missing semicolon at end of file
print "Hello,
(multiline)
World!"
