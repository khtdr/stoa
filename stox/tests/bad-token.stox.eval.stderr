Syntax error in ./bad-token.stox at line,col 3,1
3. ⚡// no such token
   ↑
Unrecognized input
Parse error in ./bad-token.stox at line,col 8,8
8. World!"
          ↑
Expected ;
